src/sec_limit_continuity.mbx

```diff
@@ -37,19 +41,20 @@
             \displaystyle \lim_{x\rightarrow c^+} f(x) = L,
           </me>
           means that given any <m>\varepsilon > 0</m>, there exists <m>\delta > 0</m> such that for all <m>x> c</m>,
-          if <m>\abs{x - c} \lt  \delta</m>, then <m>\abs{f(x) - L} \lt  \varepsilon</m>.</p>
+          if <m>\abs{x - c} \lt  \delta</m>, then <m>\abs{f(x) - L} \lt  \varepsilon</m></p>
```

Comment:
Never mind. One of us will get this when we review next. I'm going to merge now.